addon/utils/ce/raw-editor.ts

```diff
@@ -75,15 +75,20 @@ class RawEditor extends EmberObject {
   }
 
   initialize(rootNode: HTMLElement) {
-    if(this.modelSelectionTracker) {
+    if (this.modelSelectionTracker) {
       this.modelSelectionTracker.stopTracking();
     }
     this.registeredCommands = new Map<string, Command>();
     this.model = new Model(rootNode);
     this.modelSelectionTracker = new ModelSelectionTracker(this.model);
     this.modelSelectionTracker.startTracking();
+    this.rootNode.addEventListener("focus", () => {
```

Comment:
it's yet another listener we don't remove... though I guess it's removed when the element is removed :thinking: